config/kubernetes/production/service.yml

```diff
@@ -12,3 +12,18 @@ spec:
     targetPort: 3000
   selector:
     app: review-criminal-legal-aid-web-production
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: prometheus-service-production
+  namespace: review-criminal-legal-aid-web-production
```

Comment:
@mseedat-moj The `namespace` metadata here is wrong, it should be `laa-review-criminal-legal-aid-production`
Sorry I missed this on the PR review...

Same in the staging/service.yml, needs to be `laa-review-criminal-legal-aid-staging`